DATA/production/common/getStat.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# script to extract statistics from job
+
+PVs=0
+CTFsFilesProcessedMsg=$(grep "CTFReader stops processing," $1)
+nCTFsFilesOK=`echo $CTFsFilesProcessedMsg | sed 's/.*processing, \([0-9]*\) .*/\1/'`
+nCTFsFilesInspected=`grep alien_cp $1 | grep -v ALARM | grep -v non-zero | wc -l`
+nCTFsProcessed=`grep "Read CTF" $1 | tail -1 | sed 's/^.*Read CTF \([0-9]*\).*$/\1/'`
+while read -r line; do
+  currentPVs=`echo $line | sed 's/^.*Found \([0-9]*\) PVs.*/\1/'`
```

Comment:
Would not this work only for the lines which contain the vertexer report? I would add at least
```
currentPVs=`echo $lll  | grep -P "Found ([0-9]*) PVs" | sed 's/^.*Found \([0-9]*\) PVs.*/\1/'`
```

Reply:
I am not sure what you mean: the while loops over the result of the grep, a bit below.

Reply:
sorry, missed the grep below